config/managers/workloads.yaml

```diff
@@ -41,6 +41,12 @@ rules:
       - validatingwebhookconfigurations
     verbs:
       - "*"
+  - apiGroups:
```

Comment:
Should we now remove the `rbac.authorization.k8s.io/rolebindings` permissions?

Reply:
Done